web/src/components/RemoteContent.tsx

```diff
@@ -145,10 +149,25 @@ const RemoteContent = ({
         node.addEventListener('click', handleClick)
       }
     })
+
+    // iframe handling - remove not allowed iframe src from DOM and add tracking parameter
+    const iframes = sandBoxRef.current.getElementsByTagName('iframe')
+    Array.from(iframes).forEach((node: HTMLIFrameElement) => {
+      if (buildConfig().allowedIframeSources.some(el => node.src.indexOf(el) > 0) && sandBoxRef.current) {
+        var url = new URL(node.src)
+        url.searchParams.append('dnt', '1')
```

Comment:
Is this a convention that the do not track parameter is called `dnt` or is this just the case for vimeo? Because otherwise we need to keep that in mind for other video platforms.

Reply:
you are right this only works for vimeo, gonna add a check